.env.example

```diff
@@ -31,3 +31,6 @@ PORT=3333
+JWT_PUBLIC_KEY=-----BEGIN PUBLIC KEY-----\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA2cDFWbvxdOHzHVwFxB6G\n/l/RG+WNfKzxKbxO+WGuokMH2PHmDQgHvepe5UQ2CRTOPAeLcnvrfQbTwYjQE5MH\nyC0Z+MlUGIgENefrVguZr/eEqddajwklzt08GLt+YTSus+CQMifdV705nUB78i5N\n2Jk+aFTnc4appJdn1g8LFIggv41ob7n4LjFb3eAK0Jg8PQBrrdDunrLKMIbmaJ2b\nv0V6TPSH/z476okeG12eJDixU0BjKu5JZVWfGhLbUN3Kuz/n2VgSxEgdcSiTi4zl\nPjPzx3uxTg9r4oH3iYVDRkTJT4/nRKsuHLDb2tpPDcl9TPcjXtnw9H4lnQtWUpf+\naS69SDXe0EHYNtpuY3ldj79nI87hlEAHikpLOBwC7u4f9Ky7btZOu3rDtGjY/i4g\nkHZApt4GI/LazMwYkVhLdaPNHxWY54E7hBIqIFOIN9BKX7yt395OvngccpSGrmaV\nrfqDGGdaDI7mWXDl0JHf+CA82wAlDOR9fBz2yOFgm7S3WdKaSksZ/RRovlo1fA+/\nUcg7HY94pmOaOKTS+z3qU2zgfvhw1mBka4MoMFy+z35oZl+j7Ihd3VF0zl3lxjLb\n0ZZWTOM12Jm13WajpxWaHciIZNJlDlmLkbXW2aZR40JcyNYBxzkeENRjpukup8wg\na/aItHHyl6uFuaanqbrZkQECAwEAAQ==\n-----END PUBLIC KEY-----\n
```

Comment:
Is there a less verbose way than specifying the JWT keys like this?

Reply:
I don't think there is... this was automatically created by running the install commands on adonis-jwt 😞